### PR TITLE
image card: added a height and width prop

### PR DESCRIPTION
**Pull request by kitorer**

### CODE REVIEW

**Comment by kitorer** [comment]
lmk if you need any other changes or feel free to change it too

File: mobile/src/components/Card.tsx [hunk 1]
```diff
@@ -1,17 +1,29 @@
 import React from "react";
 import { Image } from "react-native";
 
+//fixme: I will probably have to break card up unto a sub-component like the web version and then url will be required
 interface Props {
   url?: string;
+  height?: string;
+  width?: string;
```
Comment (Rikimbili) [inline mobile/src/components/Card.tsx:8]: Why not generalize any style changes and use a single className instead?

Comment (kitorer) [reply mobile/src/components/Card.tsx:8]: I could but what if the person using it forgets height or width? This way there is some default value being used

Comment (Rikimbili) [reply mobile/src/components/Card.tsx:8]: You can still have a default width and height and let them override it using tailwind className. Any className that goes later in the string will override any previous one of the same properties

Comment (Rikimbili) [reply mobile/src/components/Card.tsx:8]: To be fair, your approach is not bad or anything. It's just that I prefer to have all the styling be one way and not have props that control basic styling behavior that can be done through className.

**Review by Rikimbili — DISMISSED** [review]

File: mobile/src/components/Card.tsx (hunk 1 above)
Comment (rob-3) [reply mobile/src/components/Card.tsx:8]: > To be fair, your approach is not bad or anything. It's just that I prefer to have all the styling be one way and not have props that control basic styling behavior that can be done through className.

Disagree, `className` is _the way_. Just do
```ts
<Image className={`h-96 w-11/12 ${className}`} />
```
The user can just override it by passing height or width properties and they don't have to learn about bespoke props.

**Review by rob-3 — CHANGES_REQUESTED** [review]
Please change it to use `className` passthrough; it is more idiomatic

**Review by rob-3 — APPROVED** [review]

**Pull request merged**